configure

```diff
@@ -1129,6 +1150,10 @@ configure_node(output)
 configure_library('zlib', output)
 configure_library('http_parser', output)
 configure_library('libuv', output)
+configure_library('libcares', output)
+# stay backwards compatible with shared cares builds
+output['variables']['node_shared_cares'] = \
+  output['variables'].pop('node_shared_libcares')
```

Comment:
We have a lot of stuff doing 2. Should we perhaps pep8 `configure` in a separate PR?